FpsCheck.py

```diff
@@ -329,8 +329,8 @@ def check_list_sites(self, primary, site_list):
                 if 'primary' not in json_schema.keys():
                     self.error_list.append(
                         "The listed associated site site did not have primary"
```

Comment:
FYI, Python supports multiline strings, which you could use instead of manually splitting/concatenating this string to avoid formatting issues. Same comment elsewhere.